webapp/src/main/java/life/qbic/views/register/UserRegistrationLayout.java

```diff
@@ -52,7 +52,7 @@ public class UserRegistrationLayout extends VerticalLayout {
   private H3 layoutTitle;
 
   public UserRegistrationLayout(@Autowired UserRegistrationHandlerInterface registerHandler) {
-    setId("register-view");
+    setId("handle-view");
```

Comment:
change it back pls, this breaks the styling!

Reply:
sorry, done